src/services/models/MediaType.ts

```diff
@@ -74,12 +74,22 @@ export class MediaTypeModel {
         );
       }
     } else if (this.schema) {
+      const sampledData = Sampler.sample(info.schema, samplerOptions, parser.spec);
+
+      if (disableDefaultSample && info.schema) {
```

Comment:
Instead of deleting the default sample, would it be easier to skip generating it?

Could you change it?